### DISCUSSION TITLE
Connecting to Google BiqQuery

### DISCUSSION THREAD

**Discussion by yrstrulysa** [dbeaver/dbeaver#15505] [Q&A] [answered]
Please assist me? How do i  connect to Google Bigquery.
I struggle to connect:
![image](https://user-images.githubusercontent.com/99739876/154077197-4d64afd0-83c8-466f-b94a-00842a80ad59.png)

**Comment by Destrolaric** [comment, marked as answer]
@yrstrulysa  Hi, I think you need to add your security key to the key path
https://m2msupport.net/m2msupport/generate-service-account-key-in-google-cloud-platform-gcp/

**Comment by yrstrulysa** [comment]
Thank you, i just linked to odbc instead of BigQuery with the correct odbc
connection installed and set-up. It worked.

**Comment by Destrolaric** [comment]
@yrstrulysa, I am glad that it worked. I am closing the ticket
